tablo/models.py

```diff
@@ -745,6 +770,11 @@ def update_feature(self, feature):
                         argument_values.append(feature['attributes'][key])
                     else:
                         argument_values.append(datetime.fromtimestamp(feature['attributes'][key] / 1000))
+                elif key in image_fields and feature['attributes'][key]:
+                    image_path = FeatureServiceLayer.create_image_path(self.service.id, "NO_PRIMARY_KEY", key)
```

Comment:
Its a placeholder. We don't have the PK yet. I create a dict entry where the key is the path with a placeholder key, which is later replaced with the real PK. The value is the actual image, which I need to create anyway to create the thumbnail.